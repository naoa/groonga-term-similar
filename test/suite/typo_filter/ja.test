plugin_register commands/term_similar

table_create Memos TABLE_NO_KEY
column_create Memos content COLUMN_SCALAR ShortText

table_create Terms TABLE_PAT_KEY ShortText \
  --default_tokenizer TokenDelimit \
  --normalizer NormalizerAuto \
  --token_filters TokenFilterTypo
column_create Terms memos_content COLUMN_INDEX|WITH_POSITION Memos content

load --table Memos
[
{"content": "今日は あめ だなぁ"}
]

table_tokenize Terms "あめだ" --mode GET
select Memos --match_columns content --query "あめだ"
