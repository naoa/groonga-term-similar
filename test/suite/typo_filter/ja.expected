plugin_register commands/term_similar
[[0,0.0,0.0],true]
table_create Memos TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Memos content COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer TokenDelimit   --normalizer NormalizerAuto   --token_filters TokenFilterTypo
[[0,0.0,0.0],true]
column_create Terms memos_content COLUMN_INDEX|WITH_POSITION Memos content
[[0,0.0,0.0],true]
load --table Memos
[
{"content": "今日は あめ だなぁ"}
]
[[0,0.0,0.0],1]
table_tokenize Terms "あめだ" --mode GET
[[0,0.0,0.0],[{"value":"あめ","position":0,"force_prefix":false}]]
select Memos --match_columns content --query "あめだ"
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        1
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "content",
          "ShortText"
        ]
      ],
      [
        1,
        "今日は あめ だなぁ"
      ]
    ]
  ]
]
